pytest_fixtures/core/contenthosts.py

```diff
@@ -243,7 +243,6 @@ def module_container_contenthost(request, module_target_sat, module_org, module_
         "rhel_version": "8",
         "distro": "rhel",
         "no_containers": True,
-        "network": "ipv6" if module_target_sat.network_type == NetworkType.IPV6 else "ipv4",
```

Comment:
Wouldn't it cause failures when `network_type: ipv6`?

Reply:
No, its not like that, if we don't pass network then it wouldn't work, so I think we're good to close this PR as network is still required by Broker